src/frontend/efiling-frontend/src/components/page/rush/rush-document-list/__snapshots__/RushDocumentList.test.js.snap

```diff
@@ -0,0 +1,86 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Is there a reason why we're including Jest Snapshots? Are they required?

Reply:
I assumed they were for code coverage, but since they're snapshots of tests, I'm actually not sure.

Reply:
In the PR you mean? We could just .gitignore them I think, they have just always been included on our frontend PRs so I didn't change it

Reply:
Yea I'd like to think about what is really REALLY required to be in the Repo. If the snapshot is not we reduce lines of code in our project.

Reply:
As long as we have snapshot tests, we will need to include these snapshots in the repo, so that the snapshot tests have .snap files to compare to when they are run as part of the git actions. Whether we want to keep the snapshot tests is another question, but for now the .snap files will need to be committed to the repo.